### DISCUSSION TITLE
Add Export Schedule to google calendar

### DISCUSSION THREAD

**Discussion by nmakarowski** [VikeLabs/courseup#282] [Ideas]
Option to add schedule times, sections, and locations to google calendar

**Comment by aomi** [comment]
@nmakarowski 
Issue #213 tracks the progress of this.
I ran into more hurdles than I expected so I wasn’t able to get it done by the start of the semester.
If you’re comfortable using Python I would suggest using https://github.com/malcolmseyd/uvic-schedule-exporter!

**Reply by aomi** [reply]
@nmakarowski this feature has been deployed now :)
